src/content/changes/highlights/2021-06-04.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2021-06-04T14:00:00
+title: Highlights for the week ending Jun 06, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+We fixed issues related to preparing for providing Management API access. These include correcting the chart name when installing an app and adding a missing import in the ingress installation step. We also upgraded the `ws` dependency to address a security vulnerability.
```

Comment:
Not sure if these are 'minor' but I think they are worth including. After we're all about iteration.